src/sinks/prometheus/collector.rs

```diff
@@ -301,6 +301,20 @@ impl StringCollector {
     }
 }
 
+fn format_tag(key: &str, mut value: &str) -> String {
+    let mut result = String::with_capacity(key.len() + value.len() + 4);
+    result.push_str(key);
+    result.push_str("=\"");
+    while let Some(quote) = value.find('"') {
```

Comment:
I think backslashes should be escaped as well. For example, `host\"1\"` should be formatted as `"host\\\"1\\\""` instead of `host\\"1\\"`

Reply:
Fixed